opencti-platform/opencti-graphql/src/modules/relationsRef/stixCyberObservable-registrationRef.ts

```diff
@@ -63,56 +63,56 @@ schemaRelationsRefDefinition.registerRelationsRef(
 );
 
 schemaRelationsRefDefinition.registerRelationsRef(ENTITY_DIRECTORY, [
-  buildRelationRef(contains, (fromType: string, toType: string) => [ENTITY_DIRECTORY, ENTITY_HASHED_OBSERVABLE_STIX_FILE].includes(toType))
+  buildRelationRef(contains, (_: string, toType: string) => [ENTITY_DIRECTORY, ENTITY_HASHED_OBSERVABLE_STIX_FILE].includes(toType))
 ]);
 schemaRelationsRefDefinition.registerRelationsRef(ENTITY_DOMAIN_NAME, [
-  buildRelationRef(resolvesTo, (fromType: string, toType: string) => [ENTITY_DOMAIN_NAME, ENTITY_IPV4_ADDR, ENTITY_IPV6_ADDR].includes(toType))
+  buildRelationRef(resolvesTo, (_: string, toType: string) => [ENTITY_DOMAIN_NAME, ENTITY_IPV4_ADDR, ENTITY_IPV6_ADDR].includes(toType))
 ]);
 schemaRelationsRefDefinition.registerRelationsRef(ENTITY_EMAIL_MESSAGE, [
-  buildRelationRef(from, (fromType: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
-  buildRelationRef(sender, (fromType: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
-  buildRelationRef(to, (fromType: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
-  buildRelationRef(cc, (fromType: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
-  buildRelationRef(bcc, (fromType: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
-  buildRelationRef(bodyMultipart, (fromType: string, toType: string) => ENTITY_EMAIL_MIME_PART_TYPE === toType),
-  buildRelationRef(rawEmail, (fromType: string, toType: string) => ENTITY_HASHED_OBSERVABLE_ARTIFACT === toType),
+  buildRelationRef(from, (_: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
+  buildRelationRef(sender, (_: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
+  buildRelationRef(to, (_: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
+  buildRelationRef(cc, (_: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
+  buildRelationRef(bcc, (_: string, toType: string) => ENTITY_EMAIL_ADDR === toType),
+  buildRelationRef(bodyMultipart, (_: string, toType: string) => ENTITY_EMAIL_MIME_PART_TYPE === toType),
+  buildRelationRef(rawEmail, (_: string, toType: string) => ENTITY_HASHED_OBSERVABLE_ARTIFACT === toType),
 ]);
 schemaRelationsRefDefinition.registerRelationsRef(ENTITY_EMAIL_ADDR, [
-  buildRelationRef(belongsTo, (fromType: string, toType: string) => ENTITY_USER_ACCOUNT === toType)
+  buildRelationRef(belongsTo, (_: string, toType: string) => ENTITY_USER_ACCOUNT === toType)
```

Comment:
Since the Checker type is only used in this file, maybe we could update it by removing `fromType`  from the function arguments ?